test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -467,6 +476,7 @@ class test_uct_cm_sockaddr : public uct_test {
     entity                 *m_server;
     entity                 *m_client;
     bool                   m_reject_conn_request;
+    bool                   m_switch_cm;
```

Comment:
seems like this var is not needed any more?

Reply:
right, it's a leftover. will remove.